features/qualifying_earnings_tooltip.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Qualifying earnings tooltip
+  As a user of the WPCC Tool
+  In order to gain a clear understanding of financial jargons
+  I want to be able to read further information through tooltips
+
+  Background:
+    Given I am on the Your Details step
+    When I fill in my details
+
+  # YourContribution page
+  Scenario: make minimum contribution
+    And I choose to make minimum contributions
+    And I click the Next button
+    Then I should see a tooltip next to the qualifying earnings text
+
+  # YourContribution page
+  Scenario: make full contribution
+    And I choose to make full contributions
+    And I click the Next button
+    Then I should not see a tooltip next to the qualifying earnings text
+
+  # YourResults page
+  Scenario: make minimum contribution
+    And I choose to make minimum contributions
+    # move to YourContribution page
+    And I click the Next button
```

Comment:
Is it possible to amend these steps in any way so the comments aren't necessary?

Reply:
@margOnline @davidtrussler I will look into this